keg_elements/tests/test_forms/test_form.py

```diff
@@ -92,6 +92,39 @@ def test_blank_choice_coerce(self):
         # make sure we can override the int helper by passing through an explicit coerce
         self.assert_valid(**{'letter2': 'a', 'numstr': '1'})
 
+    def test_db_enum_default_choices(self):
+        class ThingForm(ModelForm):
+            class Meta:
+                model = ents.Thing
+                csrf = False
+
+            class FieldsMeta:
+                __default__ = FieldMeta
+
+        form = ThingForm()
+        assert form.status.choices == [
+            (ents.ThingStatus.open, 'Open'),
+            (ents.ThingStatus.closed, 'Closed'),
+        ]
+        assert form.status.coerce('open') is ents.ThingStatus.open
+
+    def test_db_enum_override_choices(self):
+        class ThingForm(ModelForm):
+            class Meta:
+                model = ents.Thing
+                csrf = False
+
+            class FieldsMeta:
+                __default__ = FieldMeta
+                status = FieldMeta(choices=[('1', 'Foo'), ('2', 'Bar')], coerce=str)
```

Comment:
Why is the `coerce=str` param needed here?  From a DX perspective, it seems unexpected.

Reply:
It isn't necessary for the form operation. I was just passing a trivial function through to make sure it overrode the default coerce method.